src/Symfony/Component/Form/Button.php

```diff
@@ -293,19 +277,15 @@ public function addError(FormError $error)
     }
 
     /**
-     * Unsupported method.
-     *
-     * @return bool Always returns true
+     * {@inheritdoc}
```

Comment:
why changing all this? please only do fixes. this does not fix anything.

Reply:
This fixes the inconsistency in the phpdoc. We do satisfy the contract in this method, so why saying that it is not supported? That's my point. I know that some methods are not applicable to buttons. But this ones actually work according to the interface.

Reply:
it's not meant to be called because it does not make sense for buttons.